### PR TITLE
Website content change

### PR DESCRIPTION
**Pull request by objmagic**
minor word change, bumping version...

### CODE REVIEW

File: website/content/docs/operators/deployment/schedulers/aurora-local-setup.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 title: Setting up Heron with Aurora Cluster Locally on Linux
 ---
 
-It is possible to setup a Heron with a locally running Apache Aurora cluster. 
+It is possible to setup a Heron with a locally running Apache Aurora cluster.
```
Comment (lucperkins) [inline website/content/docs/operators/deployment/schedulers/aurora-local-setup.md]: Should be either just "Heron" or "a Heron cluster," not "a Heron"

File: website/content/docs/operators/deployment/schedulers/aurora-local-setup.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
 
-You first need to setup Apache Aurora locally. More detailed description of the 
+You first need to setup Apache Aurora locally. More detailed description of the
 following steps can be found in [A local Cluster with Vagrant](http://aurora.apache.org/documentation/latest/getting-started/vagrant/)
- 
+
 ### Step 1: Install VirtualBox and Vagrant
 
-Download and install VirtualBox and Vagrant in your machine. If vagrant is successfully 
+Download and install VirtualBox and Vagrant in your machine. If vagrant is successfully
```
Comment (lucperkins) [inline website/content/docs/operators/deployment/schedulers/aurora-local-setup.md]: on your machine

File: website/content/docs/operators/deployment/schedulers/aurora-local-setup.md (hunk 1 above)
Comment (objmagic) [inline website/content/docs/operators/deployment/schedulers/aurora-local-setup.md]: fixed via https://github.com/twitter/heron/pull/1226/commits/c0b26aa8e841acce182ac804cb07f1cb5b237e74

File: website/content/docs/operators/deployment/schedulers/aurora-local-setup.md (hunk 2 above)
Comment (objmagic) [inline website/content/docs/operators/deployment/schedulers/aurora-local-setup.md]: fixed via https://github.com/twitter/heron/pull/1226/commits/c0b26aa8e841acce182ac804cb07f1cb5b237e74

**Pull request merged**